web3/manager.py

```diff
@@ -117,6 +118,100 @@ def apply_null_result_formatters(
         return response
 
 
+def _validate_subscription_fields(response: RPCResponse) -> None:
+    params = response["params"]
+    subscription = params["subscription"]
+    if not isinstance(subscription, str) and not len(subscription) == 34:
+        _raise_bad_response_format(
```

Comment:
Similar to the `user_message`, Had a thought here that we could add a `response_property` and/or `response_property_value` or something to `BadResponseFormat`. The attribute(s) would save the specific field and/or value that caused the error (invalid value or missing field)